src/Arcus.Scripting.Management/Arcus.Scripting.Management.psm1

```diff
@@ -18,10 +18,12 @@ function Remove-AzApiManagementSoftDeletedService {
     param(
         [Parameter(Mandatory = $true)][string] $Name = $(throw "Name of the API Management instance is required"),
         [Parameter(Mandatory = $false)][string] $SubscriptionId = "",
-        [Parameter(Mandatory = $false)][string] $AccessToken = ""
+        [Parameter(Mandatory = $false)][string] $EnvironmentName = "AzureCloud",
+        [Parameter(Mandatory = $false)][string] $AccessToken = "",
+        [Parameter(Mandatory = $false)][string] $ApiVersion = "2021-08-01"
```

Comment:
Ok, now I'm nitpicking 😅 . The access token and subscription ID belongs more together? Maybe the access token can be moved a place above the environment name?

Reply:
Ah, nevermind. I see that we do this the same way as with Logic Apps. My bad! It's god as it is.